<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Agrajag Level Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="TileList" name="tileList" >
       <property name="minimumSize" >
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="acceptDrops" >
        <bool>true</bool>
       </property>
       <property name="dragEnabled" >
        <bool>true</bool>
       </property>
       <property name="iconSize" >
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="resizeMode" >
        <enum>QListView::Adjust</enum>
       </property>
       <property name="spacing" >
        <number>10</number>
       </property>
       <property name="viewMode" >
        <enum>QListView::IconMode</enum>
       </property>
      </widget>
      <widget class="LevelView" name="levelView" >
       <property name="minimumSize" >
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>574</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_level" />
    <addaction name="actionLoad_tiles" />
    <addaction name="separator" />
    <addaction name="actionSave_image" />
    <addaction name="actionSave_XML" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_Qt" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionExit" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionLoad_tiles" >
   <property name="text" >
    <string>&amp;Load tiles...</string>
   </property>
  </action>
  <action name="actionAbout_Qt" >
   <property name="text" >
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionSave_image" >
   <property name="text" >
    <string>Save &amp;image</string>
   </property>
  </action>
  <action name="actionNew_level" >
   <property name="text" >
    <string>&amp;New level...</string>
   </property>
  </action>
  <action name="actionSave_XML" >
   <property name="text" >
    <string>&amp;Save XML</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TileList</class>
   <extends>QListWidget</extends>
   <header>tilelist.h</header>
  </customwidget>
  <customwidget>
   <class>LevelView</class>
   <extends>QGraphicsView</extends>
   <header>levelview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
